tests/unit/plugins/modules/monitoring/test_datadog_downtime.py

```diff
@@ -0,0 +1,219 @@
+# -*- coding: utf-8 -*-
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+from ansible_collections.community.general.plugins.modules.monitoring.datadog import datadog_downtime
+from ansible_collections.community.general.tests.unit.compat.mock import MagicMock, patch
+from ansible_collections.community.general.tests.unit.plugins.modules.utils import (
+    AnsibleExitJson, AnsibleFailJson, ModuleTestCase, set_module_args
+)
+from datadog_api_client.v1.model.downtime import Downtime
```

Comment:
If you use `pytest.importorskip` here, this shouldn't fail when the library is not installed.